R/plot.R

```diff
@@ -110,28 +116,36 @@ plot.incidence <- function(x, ..., fit = NULL, stack = is.null(fit),
   if (is.null(ylab)) {
     if (is.numeric(x$interval)) {
       if (x$interval == 1) {
-        ylab <- "Daily incidence"
+        ylab <- "daily incidence"
       } else if (x$interval == 7) {
-        ylab <- "Weekly incidence"
+        ylab <- "weekly incidence"
       } else if (x$interval == 14) {
-        ylab <- "Biweekly incidence"
+        ylab <- "semi-weekly incidence"
       } else {
-        ylab <- sprintf("Incidence by period of %d days",
+        ylab <- sprintf("incidence by period of %d days",
                         x$interval)
       }
     } else if (is.character(x$interval)) {
-      x$interval <- gsub("^([a-z]+?)s*$", "\\1", x$interval)
-      ylab <- switch(x$interval,
```

Comment:
Here, the `ylab` string can be used for showing the bin sizes in the print info.

Reply:
I'm not sure I follow. I've updated the ylab to take into account intervals like "2 months".